src/rosecommon/packets/srv_damage.xml

```diff
@@ -1,6 +1,26 @@
 <packet ePacketType = "PAKWC_DAMAGE">
+    <includeXml path = "item.xml" />
+
     <element name = "attackerId" type = "uint16_t" />
     <element name = "defenderId" type = "uint16_t" />
     <element name = "damage" type = "uint32_t" />
-    <element name = "action" type = "uint32_t" />
+    <element name = "action" type = "DamageAction" />
```

Comment:
`action` is a bitfield. This may cause issues since this value can be a combo of different values that would not match the enum defined below.